src/status_im2/contexts/chat/home/chat_list_item/view.cljs

```diff
@@ -208,16 +208,40 @@
      {:color color
       :size  :medium}]))
 
+(defn notification
```

Comment:
This is the main component added:
it handles which badge show. the previous implementation was not taking into account `unviewed-mentions-count` for group chats